assignment2/code/security.py

```diff
@@ -0,0 +1,18 @@
+from werkzeug.security import safe_str_cmp
+
+from models.customer_model import CustomerModel
+from models.visitor_model import VisitorModel
+
+
+def authenticate(username,password):
+    # not ablt to provide validation by id
+    user= CustomerModel.find_by_username(username)
+    if user and safe_str_cmp(user.password,password) :
```

Comment:
what if user is not available

Reply:
Currently it displays a message of invalid credentials.

I wanted to implement something like this
    # if user is None:
    #     return {"message":f"User with this name {username} doesnot exists. Please signup first"}
    # if not safe_str_cmp(user.password,password):
    #     return {"message":f"Invalid Password"}
    
    But this was not working